packages/spark/base/inputs/_input-container.scss

```diff
@@ -11,3 +11,248 @@
 .sprk-b-InputContainer__visibility-toggle {
   margin: $sprk-visibility-control-margin;
 }
+
+// Applies to any huge input
+.sprk-b-InputContainer--huge .sprk-b-TextInput,
+.sprk-b-InputContainer--huge .sprk-b-Select {
+  position: relative;
+  transition: $sprk-input-huge-transition;
+  transition-delay: $sprk-input-huge-transition-delay;
+  height: $sprk-input-huge-height;
+  border-width: $sprk-input-huge-border-width;
+  border-radius: $sprk-text-input-border-radius;
+  font-size: $sprk-input-huge-font-size;
+  padding: $sprk-input-huge-padding;
+  box-shadow: $sprk-input-huge-box-shadow;
+}
+
+// Adjust for select open icon
+.sprk-b-InputContainer--huge .sprk-b-Select {
+  padding-right: $sprk-select-input-huge-padding-right;
+}
+
+.sprk-b-InputContainer--huge .sprk-b-TextInput:focus,
+.sprk-b-InputContainer--huge .sprk-b-Select:focus {
+  border-width: $sprk-input-huge-focus-border-width;
```

Comment:
Are these vars unique and not shared with standard sized inputs?

Reply:
Yes, they're supposed to be able to be set differently.